#!/bin/sh /usr/share/dpatch/dpatch-run
## 202-magic-update-awk.dpatch by Werner Fink <werner@suse.de>
##
## DP: Updates (and re-enables) awk magic.

@DPATCH@

diff -Naurp file-4.24.orig/magic/Magdir/commands file-4.24/magic/Magdir/commands
--- file-4.24.orig/magic/Magdir/commands	2008-05-04 14:12:49.000000000 +0000
+++ file-4.24/magic/Magdir/commands	2008-05-21 09:36:15.000000000 +0000
@@ -49,6 +49,7 @@
 # update to distinguish from *.vcf files
 # this is broken because postscript has /EBEGIN{ for example.
 #0	search/Bb	BEGIN {			awk script text
+0	regex		=^\\s*BEGIN\\s*[{]	awk script text
 
 # AT&T Bell Labs' Plan 9 shell
 0	string/b	#!\ /bin/rc	Plan 9 rc shell script text executable
